CalculatorTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using static Calculator.Program;
+
+namespace CalculatorTests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [Test]
+        public void AddOneNumber()
+        {
+            int answer = CalculateAnswer("+", new[] {5});
+            Assert.AreEqual(5, answer);
+        }
+
+        [Test]
+        public void AddTwoNumbers()
```

Comment:
Again using some `TestCase`'s here would be nice. You would also want to parameterise the outputs, as well as the inputs to `CalculateAnswer`, because different operators are going to produce different outputs. I'm thinking something like:

```C#
[TestCase("+", 2, 3, 5)]
[TestCase("-", 10, 3, 7)]
[TestCase("*", 18, 8, 144)]
[TestCase("/", 768, 16, 48)]
public void OperateOnTwoNumbers(string operator, int v1, int v2, int expectedAnswer)
{
    int actualAnswer = CalculateAnswer(operator, new[] {v1, v2});
    Assert.AreEqual(expectedAnswer, actualAnswer);
}
```